docs/apis-tools/zeebe-api/gateway-service.md

```diff
@@ -363,8 +366,17 @@ message CreateProcessInstanceRequest {
   // will start at the start event. If non-empty the process instance will apply start
   // instructions after it has been created
   repeated ProcessInstanceCreationStartInstruction startInstructions = 5;
+
   // the tenant id of the process definition
   string tenantId = 6;
+
+  // a reference key chosen by the user and will be part of all records resulted from this operation
+  optional uint64 operationReference = 7;
```

Comment:
This is not related to my changes but it's present in the API spec and not yet mirrored here in the docs repo. I included it in this PR because otherwise field numbers would be messed up